nodes/core/core/58-debug.js

```diff
@@ -53,7 +76,15 @@ module.exports = function(RED) {
                     }
                 }
                 if (this.active) {
-                    sendDebug({id:node.id,z:node.z,name:node.name,topic:msg.topic,property:property,msg:output,_path:msg._path});
+                    if (this.tosidebar == true) {
+                        sendDebug({id:node.id, z:node.z, name:node.name, topic:msg.topic, property:property, msg:output, _path:msg._path});
+                    }
+                    if (this.tostatus === true) {
+                        var st = util.inspect(output);
+                        if (st.length > 32) { st = st.substr(0,32) + "..."; }
+                        node.oldStatus = {fill:colors[node.severity], shape:"dot", text:st};
+                        node.status(node.oldStatus);
```

Comment:
Where did we get to with throttling of status messages to prevent overwhelming the websocket? This change will potentially double the messages going over comms... want to make sure it isn't going to cause problems if we forget to do something in comms.

Reply:
Yes I have that as well - didn't want to bite off too much in one go. Can add that in if you wish.

Reply:
Have added the comms part - but currently that breaks the tests - as it now sends and array of objects and not just a single object. As a result it also now adds timing requirements to teh tests - as it batches messages over 50ms... so before I rework tests - can you see if the approach is OK in the first place ?